script/openqa-bootstrap

```diff
@@ -23,6 +23,11 @@ if [ "$(uname -m)" = "aarch64" ]; then
     zypper -n install --no-recommends qemu-uefi-aarch64
 fi
 
+# this was split into a separate package on newer dist versions - so install it if available
+if zypper -n search -x qemu-hw-display-virtio-gpu-pci ; then
```

Comment:
I Think this should be negated with a `!`:
`if ! zypper -n search -x qemu-hw-display-virtio-gpu-pci ; then`